katz/src/main/kotlin/katz/arrow/FunctionK.kt

```diff
@@ -16,7 +16,7 @@
 
 package katz
 
-interface FunctionK<F, G> {
+interface FunctionK<in F, out G> {
```

Comment:
Not saying we don't just curious as they FunctionK in Scala does not need type variance over it's args